### DISCUSSION TITLE
Bunch of Ideas #4

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#6246] [Ideas] [answered]
Better Project Notes 
 
I don't know if ScratchAddons could do this but maybe like make an addon that could port options from Scratch Forums to Project Notes so it could include (but isn't limited to) Bold, Italics, or Underlined Text or put Bullet Points.

**Comment by BroJac5246** [comment, marked as answer]
Why did you add a frown reaction to your own post?

Unfortunately, I don't think that this is possible because people without Scratch Addons wouldn't be able to see the stylization.

**Reply by SpryzenR** [reply]
I added it because I didn't know if it would be possible
